src/lae_site/handlers/s4_signup_style.py

```diff
@@ -23,4 +23,4 @@ def render_GET(self, request):
             S4_SIGNUP_STYLE_COOKIE,
             request.args.get(b"style", [b"email"])[0]
         )
-        return redirectTo(b"/s4-subscription-form", request)
+        return request
```

Comment:
Hm.  What does this do now?

Reply:
should return the request as the client instead of redirecting to  `s4-subscription-form` with it